productize-the-model.adoc

```diff
@@ -114,12 +127,7 @@ else:
     CSV_FILE = 'creditcard.csv'
 ----
 
-We have prepared the refactored model at `/projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.ans.py`. Copy it to `/projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.py`
-
-[source,bash,role="copypaste"]
-----
-cp /projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.ans.py /projects/{{USER_MODEL_REPO_NAME}}/src/train/lr.py
-----
+Note that the code directly above this is not meant to be run but serves as an illustration for the user to take note of the versioning of the data as well as introduce the use of a bigger training file in the pipeline.
```

Comment:
I prefer top down to refer a block of text to be consistent..  You could just add this into the source code text as comments `#`. 

@eanylin ?